### PR TITLE
Bugfix/401 textarea newline strip

### PR DESCRIPTION
**Pull request by btry**
see #401 

To be cherry-picked into develop

### CODE REVIEW

File: setup.php [hunk 1]
```diff
@@ -177,17 +177,19 @@ function plugin_init_formcreator ()
  * @param  String    $string  The string to encode
  * @return String             The encoded string
```
Comment (orthagh) [inline setup.php]: we miss a docblock comment

File: inc/fields/textarea-field.class.php [hunk 2]
```diff
@@ -21,7 +21,7 @@ public function displayField($canEdit = true)
          if ($CFG_GLPI["use_rich_text"]) {
             echo plugin_formcreator_decode($this->getAnswer());
          } else {
-            echo nl2br($this->getAnswer());
+            echo str_replace(array("\\r\\n", "\\r", "\\n"), "<br />", $this->getAnswer());
```
Comment (orthagh) [inline inc/fields/textarea-field.class.php]: i'm not very pleased by this change.
If possible, we should store the data without double antislash.
nl2br is a good function, imo

**Comment by btry** [comment]
An other proposal here #410

**Comment by orthagh** [comment]
replaced by #410

**Pull request closed without merging**